test/e2e/provisioning/testdata/machine-aws.yaml

```diff
@@ -13,9 +13,9 @@ spec:
       region: "eu-west-2"
       availabilityZone: "eu-west-2a"
       vpcId: "vpc-40827629"
-      instanceType: "t2.micro"
+      instanceType: "t2.medium"
```

Comment:
may i know why we want to use bigger machines ?

Reply:
Too small machines are one of the reasons we run into timeouts during the e2e tests

Reply:
that's interesting, bigger machines are provisioned faster :)

Reply:
do you want to change the value for the others ? (`vsphere` etc.)

Reply:
I'm not 100% convinced yet if it helps but definitly worth a try.

Reply:
Well I don't think small machines were the reason why we ran into timeouts because most of the time the e2e tests passed, just sometimes they would fail because of outliers.

But bigger machines definitely improve the speed, allthought I assume there is some threshold at which it doesn't make any difference anymore (e.G. at vsphere 2 cpu was much faster than one, but I doubt if 4cpus would make it faster)